### PR TITLE
Using native Ember trackpad scroll emulator

### PR DESCRIPTION
**Pull request by taras**
Using a version of ember-scroll that's rewritten without dependency on trackpad-scroll-emulator jQuery Plugin.

### CODE REVIEW

File: addon/styles/addon.css [hunk 1]
```diff
@@ -35,14 +35,16 @@
 }
 
 .ember-light-table .lt-body-wrap {
-  flex: 1 0 auto;
   display: flex;
   flex-direction: column;
+
```
Comment (offirgolan) [inline addon/styles/addon.css]: Please remove new line

File: addon/styles/addon.css [hunk 2]
```diff
@@ -35,14 +35,16 @@
...
+  flex: 1 0 auto;
 }
 
 .ember-light-table .lt-scrollable {
   width: 100%;
-  height: 100%;
+
```
Comment (offirgolan) [inline addon/styles/addon.css]: remove new line

**Comment by taras** [comment]
@offirgolan done!

**Comment by offirgolan** [comment]
@taras how come you removed ember-scrollable from `peerDependencies`?

**Comment by taras** [comment]
@offirgolan it was necessary before because ember-cli-node-assets requires this when used with 3 level deep dependencies (App -> ember-light-table -> ember-scrollable -> trackpad-scroll-emulator). 

I eliminated `trackpad-scroll-emulator` as a dependency by rewriting it in Ember and moving all of the dependencies directly into `ember-scrollable` library.

**Comment by taras** [comment]
This is what the Ember version looks like https://github.com/alphasights/ember-scrollable/blob/master/addon/mixins/scrollable.js

**Comment by offirgolan** [comment]
@taras ah thats right. Awesome work :D

**Comment by taras** [comment]
Thanks :)

**Pull request merged**